### PR TITLE
Fix for heatmap not loading

### PR DESCRIPTION
**Pull request by translibrius**
I'm assuming `ImageStreamCompleter load()` from dart:ui was updated to `loadImage()` and the override stopped working.
This is my first ever pull request not sure what else to add 😆 
@tprebs

### CODE REVIEW

**Comment by JEndler** [comment]
This fixed the #15 issue for me 🔥

**Review by leewrigg — APPROVED** [review]
Congrats on your first PR @translibrius 🎉 

Although I haven't seen this bug myself, I've tested with this branch and everything seems to be working fine for me!

I have approved, but you'll need to wait for @tprebs to approve it too before you're able to merge 😄

**Comment by tprebs** [comment]
Thanks @translibrius. You were right, load was deprecated (https://docs.flutter.dev/release/breaking-changes/image-provider-load-buffer) in favour of loadImage and loadBuffer

**Review by tprebs — APPROVED** [review]

**Comment by tprebs** [comment]
Released in v0.0.7

**Pull request merged**
